images/router/haproxy/conf/haproxy-config.template

```diff
@@ -46,6 +46,9 @@
 {{- $pathRewriteTargetPattern := `^/.*$` -}}
 
 global
+{{- with $value := clipHAProxyTimeoutValue (firstMatch $timeSpecPattern (env "ROUTER_HARD_STOP_AFTER")) }}
```

Comment:
This is minor but for a future cleanup, we should probably change the error messages and code comments for `clipHAProxyTimeoutValue` to make sense for any timeout value generically (rather than just route timeout annotations).
https://github.com/openshift/router/blob/master/pkg/router/template/template_helper.go#L307